Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -41,15 +41,13 @@ def change_rpath(file, old_prefix, new_prefix)
     new_rpath = rpath.join(":")
     cmd = [patchelf, "--force-rpath", "--set-rpath", new_rpath]
 
-    if file.with_interpreter?
-      old_interpreter = Utils.safe_popen_read(patchelf, "--print-interpreter", file).strip
-      new_interpreter = if File.readable? "#{new_prefix}/lib/ld.so"
+    old_interpreter = file.interpreter
+    new_interpreter = old_interpreter && if File.readable? "#{new_prefix}/lib/ld.so"
```

Comment:
rubocop doesn't accept this, how do i clean this, be verbose?

Reply:
```suggestion
    old_interpreter = file.interpreter
    if old_interpreter.present?
       new_interpreter = if File.readable? "#{new_prefix}/lib/ld.so"
```

Reply:
i sense that `new_interpreter` won't defined every time. it may cause problem in below line where we check `old_interpreter != new_interpreter`